### PR TITLE
Improved logging and device support

### PR DESCRIPTION
**Pull request by tcsh4ck**

### CODE REVIEW

File: simbiota/src/detection_system.rs [hunk 1]
```diff
@@ -74,15 +74,6 @@ pub enum CommandResult {
 }
 
 static mut VERBOSE_LOG: bool = false;
```
Comment (Barrow099) [inline simbiota/src/detection_system.rs]: this could be removed too

File: simbiota/src/detection_system.rs [hunk 2]
```diff
@@ -343,49 +334,44 @@ impl DetectionSystem {
         let filename = maybe_filename.unwrap_or_else(|| "<n/a>".to_string());
         let orig_fname = filename.clone();
 
+        let logname = filename.clone();
```
Comment (Barrow099) [inline simbiota/src/detection_system.rs]: IMO this is unnecessary as the log macros try to take values by ref

**Review by Barrow099 — CHANGES_REQUESTED** [review]

File: simbiota/src/detection_system.rs (hunk 1 above)
Comment (tcsh4ck) [reply simbiota/src/detection_system.rs]: done

File: simbiota/src/detection_system.rs (hunk 2 above)
Comment (tcsh4ck) [reply simbiota/src/detection_system.rs]: done

**Review by Barrow099 — APPROVED** [review]
LGTM

**Pull request merged**